app/screens/settings/notifications/send_test_notification_notice.test.tsx

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {sendTestNotification} from '@actions/remote/notifications';
+import {logError} from '@app/utils/log';
+import {tryOpenURL} from '@app/utils/url';
+import {act, fireEvent, renderWithEverything} from '@test/intl-test-helper';
+import TestHelper from '@test/test_helper';
+
+import SendTestNotificationNotice from './send_test_notification_notice';
+
+import type Database from '@nozbe/watermelondb/Database';
+
+const version = '10.3.0';
+const oldVersion = '9.2.1';
+
+jest.mock('@utils/url', () => {
+    return {
+        tryOpenURL: jest.fn(),
+    };
+});
+
+jest.mock('@actions/remote/notifications', () => {
+    return {
+        sendTestNotification: jest.fn(),
+    };
+});
+
+jest.mock('@utils/log', () => {
+    return {
+        logError: jest.fn(),
+    };
+});
+
+const mockedSendTestNotification = jest.mocked(sendTestNotification);
+
+const allTimerExceptSetTimeout = [
+    'Date' as const,
+    'hrtime' as const,
+    'nextTick' as const,
+    'performance' as const,
+    'queueMicrotask' as const,
+    'requestAnimationFrame' as const,
+    'cancelAnimationFrame' as const,
+    'requestIdleCallback' as const,
+    'cancelIdleCallback' as const,
+    'setImmediate' as const,
+    'clearImmediate' as const,
+    'setInterval' as const,
+    'clearInterval' as const,
+    'clearTimeout' as const,
+];
+
+describe('SendTestNotificationNotice', () => {
+    let database: Database;
+
+    beforeAll(async () => {
+        const server = await TestHelper.setupServerDatabase();
+        database = server.database;
+    });
+
+    it('should match snapshot', () => {
+        const wrapper = renderWithEverything(<SendTestNotificationNotice serverVersion={version}/>, {database});
+        expect(wrapper.toJSON()).toMatchSnapshot();
+    });
+
+    it('should show nothing on older versions', () => {
+        const wrapper = renderWithEverything(<SendTestNotificationNotice serverVersion={oldVersion}/>, {database});
+        expect(wrapper.toJSON()).toMatchSnapshot();
```

Comment:
nit: I'm not a big fan of toMatchSnapshot(), and I believe, this can be checked with

```
  expect(wrapper.toJSON()).toBeNull()
```

Or, can do both. (0/5)